File: Primero_DAM/PROG/TercerTrimestre/src/Tema10Ficheros/Relacion1/Contacto.java
package Tema10Ficheros.Relacion1;

public class Contacto {
	// Atributos
	private String nombre;
	private int numero;
	
	// Constructores
	
	public Contacto() {
		
	}
	
	public Contacto(String nombre, int numero) {
		this.nombre = nombre;
		this.numero = numero;
	}

	//Getters and Setters
	
	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public int getNumero() {
		return numero;
	}

	public void setNumero(int numero) {
		this.numero = numero;
	}

	//Metodos
	@Override
	public String toString() {
		return "Contacto [Nombre: " + nombre + " Numero: " + numero + "]";
	}
	
	
}

File: Segundo_DAM/DEIN/PrimerTrimestre/ProyectosCSharp/BateriaRefuerzos/BateriaRefPractica02_ISM/BateriaRefPractica02_ISM/Contacto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BateriaRefPractica02_ISM
{
    internal class Contacto
    {
        // Atributos
        private String nombre;
        private int telefono;
        
        // Constructores
        public Contacto(string nombre, int telefono)
        {
            this.Nombre = nombre;
            this.Telefono = telefono;
        }

        // Getters and Setters
        public string Nombre { get => nombre; set => nombre = value; }
        public int Telefono { get => telefono; set => telefono = value; }

    }
}
